### PR TITLE
add method for triggering test event for campaign

### PR DESCRIPTION
**Pull request by vrabeshko**

### CODE REVIEW

**Comment by vrabeshko** [comment]
[M - Create Test subscription that can be triggered by SDK](https://trello.com/c/XUSlAfoz/1113-m-create-test-subscription-that-can-be-triggered-by-sdk)

File: src/XcooBee/Core/Api/System.php [hunk 1]
```diff
@@ -198,6 +198,33 @@ public function getEvents($config = [])
...
+     *
+     * @param string $type
+     * @param array $config
+     *
+     * @return Response
+     * @throws XcooBeeException
+     */
+    public function triggerEvent($type, $config = [])
+    {
+        $campaignId = $this->_getCampaignId(null, $config);
```
Comment (Bilal-S) [inline src/XcooBee/Core/Api/System.php]: For `type` we should check the list of valid events locally before sending to XcooBee backend.

File: README.md [hunk 2]
```diff
@@ -339,6 +339,27 @@ standard JSON response object
     - data will contain the number of deleted subscriptions
 - status 400 if error
 
+## triggerEvent(type, [config])
```
Comment (Bilal-S) [inline README.md]: I will rewrite this after we finalize changes.

**Comment by Bilal-S** [comment]
Cool. I will approve.

**Pull request merged**